tests/web/test_outputs.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from unittest.mock import MagicMock, patch
+
+pytestmark = [pytest.mark.bdb, pytest.mark.usefixtures('inputs')]
+
+UNSPENTS_ENDPOINT = '/api/v1/outputs/'
+
+
+def test_get_outputs_endpoint(client, user_pk):
+    m = MagicMock()
+    m.to_uri.side_effect = lambda s: s
+    with patch('bigchaindb.core.Bigchain.get_outputs_filtered') as gof:
+        gof.return_value = [m, m]
+        res = client.get(UNSPENTS_ENDPOINT + '?public_key={}'.format(user_pk))
+    assert res.json == ["..", ".."]
+    assert res.status_code == 200
+    gof.assert_called_once_with(user_pk, True)
+
+
+def test_get_outputs_endpoint_unspent(client, user_pk):
+    m = MagicMock()
+    m.to_uri.side_effect = lambda s: s
+    with patch('bigchaindb.core.Bigchain.get_outputs_filtered') as gof:
```

Comment:
Sorry. I should have been more explicit here.

As far as I can tell you're using it absolutely correct, but you could save a few lines by using the patch operator. Totally up to you if you wanna go for that though.